example/bzlmod/BUILD

```diff
@@ -0,0 +1,20 @@
+load("@gazelle//:def.bzl", "gazelle", "gazelle_binary")
+
+gazelle_binary(
+    name = "gazelle_cc",
+    languages = [
+        "@gazelle//language/proto",
+        "@gazelle_cc//language/cc",
+    ],
+)
+
+gazelle(
+    name = "gazelle",
+    gazelle = ":gazelle_cc",
+)
+
+# gazelle:cc_group unit
+# gazelle:resolve cc gtest/gtest.h @googletest//:gtest_main
+
+# TODO Workaround to bug in proto gazelle
+# gazelle:map_kind cc_proto_library cc_proto_library @com_google_protobuf//bazel:cc_proto_library.bzl
```

Comment:
Is `@com_google_protobuf` necessary here? Could it work as `@protobuf`? If so, then you could get rid of `repo_name = "com_google_protobuf"` in `MODULE.bazel`.

Reply:
Originally there was some issue with gazelle not finding well known deps of .proto file when using `@protobuf`, but it worked with the workaround. 

```
    deps = [
        "@protobuf//:timestamp_proto",
        "@protobuf//:type_proto",
    ],
```
I've tried it now, and the deps are correctlly resolved, so this workaround can be removed